commit a213c79fe1ef64b7bd0e58bd7d383832d8afe513
Author: Theodore Ts'o <tytso@mit.edu>
Date:   Sun Feb 5 01:26:48 2017 -0500

    ext4: return EROFS if device is r/o and journal replay is needed
    
    commit 4753d8a24d4588657bc0a4cd66d4e282dff15c8c upstream.
    
    If the file system requires journal recovery, and the device is
    read-ony, return EROFS to the mount system call.  This allows xfstests
    generic/050 to pass.
    
    Signed-off-by: Theodore Ts'o <tytso@mit.edu>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1283_a213c79fe1ef64b7bd0e58bd7d383832d8afe513.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1283_a213c79fe1ef64b7bd0e58bd7d383832d8afe513.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
